### PR TITLE
Pandas 2.0.3

### PR DESCRIPTION
**Pull request by ryanskeith**
# Log
- Update version and sha
- Updated dependencies
- Added notes to cite which dependencies corresponded with which add ons

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,62 +29,81 @@ requirements:
...
-    - pandas-gbq >=0.15.0
-    - s3fs >=2021.08.0
+    # html, xml
+    - beautifulsoup4 >=4.9.3
+    - html5lib >=1.1
+    - lxml >=4.6.3
+    # compression
     - brotli >=0.7.0
     - python-snappy >=0.6.0
     - zstandard >=0.15.2
```
Comment (cbouss) [inline recipe/meta.yaml:95]:
```suggestion
    - zstandard >=0.15.2
    # clipboard
    - pyqt >=5.15.1,<6
    - qtpy >=2.2.0
```
https://github.com/pandas-dev/pandas/blob/v2.0.3/pyproject.toml#L79

File: recipe/meta.yaml [hunk 2]
```diff
@@ -29,62 +29,81 @@ requirements:
...
 test:
+  source_files:
+    - pyproject.toml
   requires:
     - pip
-      # v1.5.3: project's code repo specifies >=5.5.3 but they suggest to use >=6.13.0 here: https://pandas.pydata.org/docs/getting_started/install.html#running-the-test-suite
-    - hypothesis >=6.13.0
-    - pytest >=6.0
-    - pytest-xdist >=1.31
+    - hypothesis >=6.34.2
```
Comment (cbouss) [inline recipe/meta.yaml:103]: It doesn't look like we are running the test suite.
In that case we might comment out these dependencies.

Comment (skupr-anaconda) [reply recipe/meta.yaml:103]: As this is a **major** version bump could we include a test suite to check if there might be errors we are not aware?

Comment (ryanskeith) [reply recipe/meta.yaml:107]: I think you are right @skupr-anaconda , Let's test as it is a major release.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
The build for `Pandas 2.0.3` seems to fail in `Windows` and in `Linux s390x`
  It might be a good idea to see if the issues could be resolved before merging to defaults.

**Pull request merged**